CHANGELOG.md

```diff
@@ -3,14 +3,47 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## v1.0dev - [date]
+## [[1.0.0](https://github.com/sanger-tol/blobtoolkit/releases/tag/1.0.0)] – ??? - [2023-04-06]
 
-Initial release of nf-core/blobtoolkit, created with the [nf-core](https://nf-co.re/) template.
+### Enhancements & fixes
 
-### `Added`
+Initial release of sanger-tol/blobtoolkit :tada
 
-### `Fixed`
+This release marks the point where the pipeline was moved from Snakemake at [blobtoolkit/blobtoolkit](https://github.com/blobtoolkit/blobtoolkit) over to Nextflow DSL2 at [sanger-tol/blobtoolkit](https://github.com/sanger-tol/blobtoolkit).
 
-### `Dependencies`
+### Parameters
 
-### `Deprecated`
+| Old parameter | New parameter |
+| ------------- | ------------- |
+|create_blobdir (blobtools replace): --evalue 1.0e-25 --hit-count 10 |  --evalue 1.0e-25 --hit-count 10 |
+| diamond_blastp: --evalue 1.0e-25  --max-target-seqs 10 --max-hsps 1 | --evalue 1.0e-25  --max-target-seqs 10 --max-hsps 1 |
+| diamond_blastx: --evalue 1.0e-25  --max-target-seqs 10              |               |
+| stats_windows: --window 0.1 --window 0.01 --window 100000 --window 1000000  |--window 0.1 --window 0.01 --window 100000 --window 1000000|
```

Comment:
I don't understand the difference between the two columns: these are the same values, right ?

Reply:
Yes, I was not sure if we should specify these parameters in the case they are the same in Snakemake and Nextflow versions. So, if they are the same they should not be included in the table, right?

Reply:
> Yes, I was not sure if we should specify these parameters in the case they are the same in Snakemake and Nextflow versions. So, if they are the same they should not be included in the table, right?

Correct 👍🏼